features/gm.ts

```diff
@@ -126,8 +126,8 @@ export class GroupMessageController implements GroupMessageGetter, GroupMessageL
 
     private async handleInvitation(event: NostrEvent<NostrKind.Group_Message>) {
         if (event.pubkey == this.ctx.publicKey.hex) {
-            return new Error("the invitation created by me.");
-        } // send by me
+            return; // ignore this case because this is an invitation created by me
```

Comment:
Because the caller will remove the event is an error is returned, we don't want the caller to treat this case as an error.